uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/components/leave-request-popup-details-tab.component.js

```diff
@@ -565,6 +585,29 @@ define([
       vm.pagination.filteredbreakdown = vm.balance.change.breakdown.slice(begin, end);
     }
 
+    /**
+     * Performs a set of manipulations to prepare all data for calculation and
+     * then triggers the calculation itself if it can be calculated
+     *
+     * @return {Promise}
+     */
+    function performBalanceChangeCalculation () {
```

Comment:
this looks much better, straight to the point (thumbsup)

Reply:
Proposed a function name, commented on a change, grated yourself, thumbed up own comment. Nice one @AkA84! :D